sdk/src/contracts/scripts/deploy-integration-registry.js

```diff
@@ -1,16 +1,30 @@
 /* eslint-disable */
 const { ethers, upgrades } = require('hardhat');
+//const pkgInfos = require('../../../../build/integrations_bucket.json');
 async function main() {
-  //const IntegrationRegistry = await ethers.getContractFactory('IntegrationRegistry');
+  const IntegrationRegistry = await ethers.getContractFactory('IntegrationRegistry');
   //const IntRegistrar = await upgrades.deployProxy(IntegrationRegistry);
```

Comment:
this code is no longer used?

Reply:
it is used, but I'm keeping it commented to prevent unwanted redeployments until we have an automated process